tests/conftest.py

```diff
@@ -596,16 +602,11 @@ def pytest_runtest_setup(item):
                 "flaky jail tests are disabled, pass '--flaky-jail' to enable them.",
                 _use_item_location=True,
             )
-    if (
-        not item.get_closest_marker("slow_test")
-        and not item.get_closest_marker("core_test")
-        and not item.get_closest_marker("flaky_jail")
-    ):
-        if not item.config.getoption("--no-fast-tests"):
-            raise pytest.skip.Exception(
-                "Fast tests are disabled, dont pass '--no-fast-tests' to enable them.",
-                _use_item_location=True,
-            )
+    if test_group_count == 0 and not item.config.getoption("--no-fast-tests"):
+        raise pytest.skip.Exception(
+            "Fast tests are disabled, dont pass '--no-fast-tests' to enable them.",
```

Comment:
This wording seems off. Regular users must have a better understanding of why they are getting an error.

Reply:
People will only see this when they pass `--no-fast-tests`. When a fast test is skipped they will see this message.

Reply:
SSH does the same thing `SSH tests are disabled, pass '--ssh-tests' to enable them."`

Reply:
> SSH does the same thing `SSH tests are disabled, pass '--ssh-tests' to enable them."`

Indeed, but it's `--ssh-tests` not `--no-ssh-tests` (to enable them)

So either the flag or the error message looks off.

Reply:
how would you like me to word the error message?

Reply:
The `--no-fast-tests` flag makes sense if the default is to run the fast tests. Otherwise, it makes sense to specify fast tests with the `--fast-tests` flag.

Reply:
How about this:

```
Fast tests have been disabled by '--no-fast-tests'.
```